mod.config

```diff
@@ -37,7 +37,8 @@ PACKAGING_WINDOWS_REGISTRY_KEY="OpenRARA2Mod"
 # Can be overwritten in user.config if you want to manage it manually/using git (e.g. submodules)
 AUTOMATIC_ENGINE_MANAGEMENT="True"
 
-AUTOMATIC_ENGINE_SOURCE="https://github.com/OpenRA/OpenRA/archive/${ENGINE_VERSION}.zip"
+# HACK HACK HACK: Use a custom engine which reverts https://github.com/OpenRA/OpenRA/pull/14043
+AUTOMATIC_ENGINE_SOURCE="https://github.com/abcdefg30/OpenRA/archive/ra2-master.zip"
```

Comment:
This doesn't file right... However I don't have a good idea on how to resolve the sound issues at the moment.